src/lib/PreviewUI.js

```diff
@@ -385,9 +392,8 @@ class PreviewUI {
             const defaultLogoEl = headerEl.querySelector(SELECTOR_BOX_PREVIEW_LOGO_DEFAULT);
             defaultLogoEl.classList.add(CLASS_HIDDEN);
 
-            const customLogoEl = headerEl.querySelector(SELECTOR_BOX_PREVIEW_LOGO_CUSTOM);
-            customLogoEl.src = logoUrl;
-            customLogoEl.classList.remove(CLASS_HIDDEN);
+            this.customLogoEl.src = logoUrl;
```

Comment:
I don't think this covers all our use cases, since we have both a "default" element and a "custom" logo element. I suggest we rearrange `setupHeader` to apply the `alt` attribute to both.